external/CMakeLists.txt

```diff
@@ -185,3 +185,43 @@ target_include_directories(breakpad_client
     PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>"
 )
+
+target_compile_features(breakpad_client PUBLIC cxx_std_11)
```

Comment:
Why was this line in your build necessary (we already define the `CMAKE_CXX_STANDARD` for our C++ dependencies)? AFAIK as I know upstream `breakpad` switched to C++17 (but it might not yet be necessary to enable in the `breakpad` client). I am more than happy to switch to a target-based setting of C++ standard enforcement, but this should happen in another PR and be applied to all targets so we can remove the global setting.